Common subdirectories: qct/doc and qct-build/doc
Common subdirectories: qct/.hg and qct-build/.hg
Common subdirectories: qct/hgext and qct-build/hgext
diff -u qct/Makefile qct-build/Makefile
--- qct/Makefile	2012-10-07 23:38:55.000000000 +0800
+++ qct-build/Makefile	2012-10-07 23:44:46.000000000 +0800
@@ -10,7 +10,7 @@
 # Revisioned files which are not distributed
 PRUNE  := .hgtags .hgignore
 
-VER := $(shell python setup.py --version)
+VER := $(shell python2 setup.py --version)
 DIST_NAME  := qct-$(VER)
 PRUNE_LIST := $(foreach file, $(PRUNE), $(DIST_NAME)/$(file))
 
@@ -24,7 +24,7 @@
 	$(RM) -r $(DIST_NAME)/
 
 ui_%.py : %.ui
-	pyuic4 $< > $@
+	python2-pyuic4 $< > $@
 
 doc/qct.1:
 	$(MAKE) -C doc
@@ -32,13 +32,13 @@
 .PHONY: install site-install mac-install
 
 install: $(UI_FILES)
-	python setup.py install --home=~
+	python2 setup.py install --home=~
 
 site-install: $(UI_FILES)
-	python setup.py install --force
+	python2 setup.py install --force
 
 egg: $(UI_FILES)
-	python setup.py bdist_egg
+	python2 setup.py bdist_egg
 
 clean:
 	$(RM) $(UI_FILES)
Common subdirectories: qct/plugins and qct-build/plugins
Common subdirectories: qct/qctlib and qct-build/qctlib
diff -u qct/setup.py qct-build/setup.py
--- qct/setup.py	2012-10-07 23:38:55.000000000 +0800
+++ qct-build/setup.py	2012-10-07 23:44:49.000000000 +0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 #
 # This is the qct setup script.
 #
Common subdirectories: qct/win32 and qct-build/win32
